EXPRESSIONS.md

```diff
@@ -1590,6 +1591,31 @@ to evaluate additional functions against.
 ${allMatchingAttributes('a.*'):count()} ):gt(3)}` returns true if there are
 more than 3 attributes whose names begin with the letter a.
 
+### reverseLookup
```

Comment:
maybe we could call this `reverseDNSLookup`?  `reverseLookup` sounds more general than it really is

Reply:
good point, I've renamed it in https://github.com/apache/nifi-minifi-cpp/pull/1510/commits/31dacdb37bcf4cdad0f32d6311a047f549920dd4